Web-Development/API-Development/level-2/api-level-2-user_manage-ment-Pranavgurung/app.py

```
@@ -0,0 +1,143 @@
+        "endpoints": {
+            "POST": "/api/users",
+            "GET": "/api/users",
+            "GET (one)": "/api/users/<id>",
+            "PUT": "/api/users/<id>",
+            "DELETE": "/api/users/<id>"
+        }
+    }), 200
+
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Ruff (0.14.2)

Use of `debug=True` in Flask app detected

(S201)